crates/metadata-server/src/lib.rs

```diff
@@ -231,23 +231,23 @@ where
 }
 
 pub async fn create_metadata_server(
-    metadata_store_options: &MetadataStoreOptions,
+    metadata_server_options: &MetadataServerOptions,
     rocksdb_options: BoxedLiveLoad<RocksDbOptions>,
     health_status: HealthStatus<MetadataServerStatus>,
     metadata_writer: Option<MetadataWriter>,
     server_builder: &mut NetworkServerBuilder,
 ) -> anyhow::Result<BoxedMetadataStoreService> {
-    match metadata_store_options.kind {
-        MetadataStoreKind::Local => local::create_server(
-            metadata_store_options,
+    match metadata_server_options.kind {
+        MetadataServerKind::Local => local::create_server(
+            metadata_server_options,
             rocksdb_options,
             health_status,
             server_builder,
         )
         .await
         .map_err(anyhow::Error::from)
         .map(|store| store.boxed()),
-        MetadataStoreKind::Raft => raft::create_server(
+        MetadataServerKind::Raft { .. } => raft::create_server(
```

Comment:
Perhaps we can call this "replicated" metadata store as well. Not in the scope of this PR but as part of polishing this up. I've not strong opinions tbh.

Reply:
Replicated works for me. Atm the name in the configuration is "embedded".